### PR TITLE
fix(l-n-b-menu-item): separate LNBMenuItem component

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA
- separated for additional usage of LNB item

### CODE REVIEW

File: apps/web/src/common/modules/navigations/lnb/modules/LNBDividerMenuItem.vue [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    </div>
+</template>
+
+<style lang="postcss" scoped>
+.l-n-b-divider-menu-item {
+    margin-top: 1.25rem;
+    margin-bottom: 1.25rem;
+    height: 0;
+}
+</style>
```
Comment (skdud4659) [inline apps/web/src/common/modules/navigations/lnb/modules/LNBDividerMenuItem.vue:17]: Q. Is there a reason for separating components even though there's only one divider?

**Review by yuda110 — APPROVED** [review]
LGTM! Could you share this on Wed?

**Review by skdud4659 — APPROVED** [review]
LGTM~!
i left a question!

File: apps/web/src/common/modules/navigations/lnb/modules/LNBDividerMenuItem.vue (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/common/modules/navigations/lnb/modules/LNBDividerMenuItem.vue:17]: There are some cases without clear usage, so I separated this for usage with spec.

**Pull request merged**